tests/integration/end2end/tc.yaml

```diff
@@ -28,8 +31,8 @@ pipeline:
     metric_fn: !Accuracy
     optimizer: !torch.Adam
       params: !@ b1[model].trainable_params
-    max_steps: 2
+    max_steps: 1
     iter_per_step: 1
-    extra_validation_metrics:
-      - !@ b1[metric_fn]
-      - !@ b1[metric_fn]
+    # extra_validation_metrics:
```

Comment:
The `extra_validation_metrics` are failing because of linking issues. @nmatthews-asapp is working on this. **Locally it also fails.**